ladybug/comfort/standard/pmv.py

```diff
@@ -0,0 +1,406 @@
+        """The percent of time that the thermal_condiiton is neutral."""
+        return self._percent_neutral
+
+    @property
+    def percent_cold(self):
+        """The percent of time that the thermal_condiiton is cold."""
+        return self._percent_cold
+
+    @property
+    def percent_hot(self):
+        """The percent of time that the thermal_condiiton is hot."""
+        return self._percent_hot
+
+    @property
+    def percent_dry(self):
+        """The percent of time that the thermal_condiiton is too dry."""
+        return self._percent_dry
+
+    @property
+    def percent_humid(self):
+        """The percent of time that the thermal_condiiton is too humid."""
+        return self._percent_humid
+
+    @property
+    def ta_adj(self):
+        """DataCollection of air temperatures that have been adjusted by the SET model
+        to account for the effect of air speed [C].
+        """
+        return self._ta_adj
+
+    @property
+    def cooling_effect(self):
+        """DataCollection of the cooling effect of the air speed in degrees Celcius.
+
+        This is the difference between the air temperature and the
+        adjusted air temperature [C].
+        """
+        return self._cooling_effect
+
+    @property
+    def heat_loss_conduction(self):
+        """DataCollection of heat loss by conduction in [W]."""
+        return self._heat_loss_conduction
+
+    @property
+    def heat_loss_sweating(self):
+        """DataCollection of heat loss by sweating in [W]."""
+        return self._heat_loss_sweating
+
+    @property
+    def heat_loss_latent_respiration(self):
+        """DataCollection of heat loss by latent respiration in [W]."""
+        return self._heat_loss_latent_respiration
+
+    @property
+    def heat_loss_dry_respiration(self):
+        """DataCollection of heat loss by dry respiration in [W]."""
+        return self._heat_loss_dry_respiration
+
+    @property
+    def heat_loss_radiation(self):
+        """DataCollection of heat loss by radiation in [W]."""
+        return self._heat_loss_radiation
+
+    @property
+    def heat_loss_convection(self):
+        """DataCollection of heat loss by convection in [W]."""
+        return self._heat_loss_convection
+
+    def ToString(self):
+        """Overwrite .NET ToString."""
+        return self.__repr__()
+
+    def __repr__(self):
+        """PMV comfort model representation."""
+        return "PMV Comfort Model: {} values".format(self._calc_length)
+
+
+class PMVComfortParameters(ComfortParameter):
+    """Parameters of PMV comfort."""
+
+    def __init__(self, ppd_comfort_thresh=None, humid_ratio_upper=None,
+                 humid_ratio_lower=None, still_air_threshold=None):
+        """Initalize PMVComfortParameters
+
+        Properties:
+            ppd_comfort_thresh
+            humid_ratio_upper
+            humid_ratio_low
+            still_air_threshold
+        """
+
+        self._ppd_thresh = ppd_comfort_thresh or 10
```

Comment:
since 0 is an acceptable value for most of these inputs except for ppd you want to change them to also accept 0.